app/src/main/java/org/oppia/android/app/walkthrough/topiclist/WalkthroughTopicListFragment.kt

```diff
@@ -31,4 +31,9 @@ class WalkthroughTopicListFragment : InjectableFragment(), TopicSummaryClickList
   override fun onTopicSummaryClicked(topicSummary: TopicSummary) {
     walkthroughTopicListFragmentPresenter.changePage(topicSummary)
   }
+
+  override fun onDestroyView() {
+    super.onDestroyView()
+    walkthroughTopicListFragmentPresenter.handleDestroyView()
```

Comment:
I think we can call `hideHeaderAndShowProgressBar ` as that was a public method?

Reply:
Done